src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -385,7 +385,6 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
                                   events: Seq[ExecutionEventEntry]) = {
     val currentSender = sender()
     val keyClone = callKey.retryClone
-    executionStore += keyClone -> ExecutionStatus.NotStarted
```

Comment:
I think you're right this is causing the error but I'm surprised that dependency exists.  I need to dig a bit more on this one...